frontend/packages/core/src/Table/table.tsx

```diff
@@ -115,6 +117,8 @@ const TableContainer = ({ children }: TableContainerProps) => (
   </MuiTableContainer>
 );
 
+export type Order = "asc" | "desc";
```

Comment:
We can probably pull this from the TableSortLabelProps and just extend it onto the props.

Reply:
in the example from the docs it was define like this, not imported from MUI library

Reply:
Which docs, material's?

Ultimately it's just one piece we don't have to manually handle since the prop exists and thats what we're passing it onto anyways. It's not a huge deal since it's only two possibilities though :)